scripts/boost.sh

```diff
@@ -26,7 +26,7 @@ INCLUDES=(
 
     iterator range config detail exception functional predef system type_traits
     core io utility filesystem mpl preprocessor smart_ptr asio date_time bind
-    regex
+    regex container_hash
```

Comment:
I don't see `container_hash` being included. Is it needed?

Reply:
Upon a second look, it appears as though the version of boost was updated. I'm curious as to what bug was fixed as it could be affecting another project of mine.

Reply:
Looks like `converter/cast.hpp` is also needed.

Reply:
no, this was an issue on my side, I deleted the corresponding commit. I had boost installed system-wide at a different version, and it got picked by the compiler instead of the packaged one.

I'll try to see how I can prevent this from happening again =)